### PR TITLE
Atomic Registry: add non existent user to project

### PR DESCRIPTION
**Pull request by sub-mod**

### CODE REVIEW

**Comment by sub-mod** [comment]
@petervo here is the change to add users who are not logged in

**Comment by petervo** [comment]
@andreasn does this fix the issue you had with dropdown.

**Comment by andreasn** [comment]
> does this fix the issue you had with dropdown.

Yep, it looks like it should now:

screenshot from 2016-03-31 12-29-15

screenshot from 2016-03-31 12-29-30

**Comment by stefwalter** [comment]
There's a failure in the tests:

```
Page error: #add_member not found phantomjs://code/phantom-lib.js 24 ph_only phantomjs://code/phantom-lib.js 33 ph_find phantomjs://code/phantom-lib.js 151 ph_is_visible 1 1 
```

**Comment by sub-mod** [comment]
not sure how rhel-atomic works and fedora atomic fails.

**Comment by stefwalter** [comment]
> not sure how rhel-atomic works and fedora atomic fails.

Yes, that's strange. Maybe a race condition?

**Comment by sub-mod** [comment]
waiting for testsuite prepare to complete

**Comment by sub-mod** [comment]
integrtion test passed
+ journalctl -o cat SYSLOG_IDENTIFIER=kernel 2>&1 | grep 'type=14.*audit' || true
ok 1 testProjectPolicy (__main__.TestRegistry)

 TESTS PASSED [44s on mysystem]

File: pkg/kubernetes/scripts/projects.js [hunk 1]
```diff
@@ -330,47 +330,59 @@
...
                 ocRole: "",
             };
             
             var namespace = fields.namespace;
-            function getPolicyBinding(namespace){
-                return kselect().kind("PolicyBinding").namespace(namespace).name(":default");
-            }
-            function getAllMembers() {
+
+            function getAllMembers(namespace) {
```
Comment (petervo) [inline pkg/kubernetes/scripts/projects.js]: Why does this function take a namespace argument? Is it used in the function?

There is also a namespace variable declared right above it, making it confusing. If you really need both I'd suggest renaming one of those variables to make it more readable and avoid potential scope issues.

File: pkg/kubernetes/scripts/projects.js [hunk 2]
```diff
@@ -384,14 +396,29 @@
...
-                    var patchObj = getPolicyBinding(namespace);
+                var memberName = $scope.select.memberName;
+                var member = $scope.select.member;
+                var memberObj, kind;
+                if (memberName && memberName === member) {
+                    //dropdown value selected
+                    memberObj = $scope.select.memberObj;
+                    memberName = memberObj.name;
+                    kind = memberObj.kind;
+                } else if(memberName && member === selectMember) {
```
Comment (stefwalter) [inline pkg/kubernetes/scripts/projects.js]: Missing space after if here.

File: pkg/kubernetes/scripts/projects.js [hunk 3]
```diff
@@ -384,14 +396,29 @@
...
+                    //input field has value
+                    kind = "User";
+                } else if(!memberName && member === selectMember) {
```
Comment (stefwalter) [inline pkg/kubernetes/scripts/projects.js]: Missing space after if here.

**Comment by stefwalter** [comment]
This seems to already be merged.

**Pull request closed without merging**